database/game_model.js

```diff
@@ -26,7 +26,11 @@ const Game = new mongoose.model('Game', gameSchema);
 
 const getSingleGame = async (gameId) => {
   try {
-    return await Game.findOne({gameId}).exec();
+    const foundGame = await Game.findOne({gameId}).exec();
+    if (foundGame.length === 0) {
+      throw err;
+    }
+    return foundGame;
   } catch (error) {
     throw error;
```

Comment:
So when that error gets thrown on line 35 I believe it will get handled in server/app.js when a request is made. I will double check but in my head line 31 will be handled by that 404 in server/app.js and line 35 will be handled by that 500 in server/app.js